### PR TITLE
WIP: add mummer as dependency to quast

### PR DESCRIPTION
**Pull request by bgruening**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by natefoo** [comment]
Quast depends on [E-MEM](http://www.csd.uwo.ca/~ilie/E-MEM/) and MUMmer if using the reference (`-R`) option. It will not use external versions of these, however. It attempts to build them at runtime in `opt/quast-4.1/libs/{E-MEM-linux,MUMmer3.223-linux}`. It may be possible to use the bioconda MUMmer with some linking tricks.

**Comment by lparsons** [comment]
I'm a bit in favor of building the packaged E-MEM since that is what quast is expecting and it's what non-bioconda users will be doing. Provides a more "standard" install, IMHO, even if it's not as "clean" from the dependency perspective.

File: recipes/quast/build.sh [hunk 1]
```diff
@@ -8,6 +8,8 @@ mkdir -p $QUAST_HOME
 
 cp -R $SRC_DIR/* $QUAST_HOME
 
+make -C $PREFIX/opt/quast-4.1/libs/E-MEM-linux
```
Comment (natefoo) [inline recipes/quast/build.sh]: Probably also need `make -C $PREFIX/opt/quast-4.1/libs/MUMmer3.23-linux` here

**Comment by bgruening** [comment]
@natefoo can you try the latest version?

**Pull request merged**